thorlcr/activities/funnel/thfunnelslave.cpp

```diff
@@ -384,7 +383,8 @@ class FunnelSlaveActivity : public CSlaveActivity
             {
                 try
                 {
-                    LookAheadTimer timer(slaveTimerStats, timeActivities);
+                    // n.b. i>0 is started asynchronously, so track look ahead time
+                    LookAheadTimer timer(slaveTimerStats, (i==0) ? false : timeActivities);
```

Comment:
minor: better to avoid constructing LookAheadTimer and coding as something like:
```
if (i == 0) // 1st input is started synchronously, so time already included in start() timing.
    startInput(i);
else
{
    LookAheadTimer timer(slaveTimerStats, timeActivities);
    startInput(i);
}
```